recipe/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if [[ `uname` == 'Darwin' ]]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.9
```

Comment:
Yeah, that is a C++11 thing. Try using the `toolchain` and then add `-std=c++11` to the `CXXFLAGS`. Please bump the build number too. Please ping me if that doesn't just work.